weave-js/src/util/llmTokenCosts.ts

```diff
@@ -0,0 +1,181 @@
+export const modelTuple = [
+  'default',
+  'gpt-4o',
+  'gpt-4o-2024-05-13',
+  'gpt-4-turbo',
+  'gpt-4-turbo-2024-04-09',
+  'gpt-4',
+  'gpt-4-32k',
+  'gpt-4-0125-preview',
+  'gpt-4-1106-preview',
+  'gpt-4-vision-preview',
+  'gpt-3.5-turbo-1106',
+  'gpt-3.5-turbo-0613',
+  'gpt-3.5-turbo-16k-0613',
+  'gpt-3.5-turbo-0301',
+  'gpt-3.5-turbo-0125',
+  'gpt-3.5-turbo-instruct',
+  'davinci-002',
+  'babbage-002',
+  'claude-3-opus-20240229',
+  'claude-3-sonnet-20240229',
+  'claude-3-haiku-20240307',
+];
+
+export type MODEL = (typeof modelTuple)[number];
```

Comment:
nit: instead of defining the modelTuple as a hard-coded list, you could get the keys from `LLM_TOKEN_COSTS`. and `export type MODEL` would be defined as a key of `LLM_TOKEN_COSTS`